<!DOCTYPE html>
<html lang="en">
  <head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Publications</title>

    
    <link href="https://fonts.googleapis.com/css2?family=Urbanist:wght@400;600;700&display=swap" rel="stylesheet">

    <link href="https://fonts.googleapis.com/css2?family=El+Messiri:wght@400;700&display=swap" rel="stylesheet">

    
    
    <link rel="stylesheet" href="//localhost:1313/css/custom.min.e302af2495ec0cf54b4b3b86f9780c56f1959687844a7ac972d3b837aec4da54.css">    
  </head>
  <body>
    <header class="site-header">
  <div class="site-title">
    <a href="https://naclist.github.io/naclist-portfolio/">Naclist</a>
  </div>

  
  <div class="site-nav-wrapper">
    <nav class="site-nav">
      
        <a href="//localhost:1313/"
           class="">
          Home
        </a>
      
        <a href="//localhost:1313/publications/"
           class="active">
          Publications
        </a>
      
        <a href="//localhost:1313/skills/"
           class="">
          Skills
        </a>
      
        <a href="//localhost:1313/ambitions/"
           class="">
          Ambitions
        </a>
      
        <a href="//localhost:1313/blogs/"
           class="">
          Blogs
        </a>
      
        <a href="//localhost:1313/arts/"
           class="">
          Arts
        </a>
      
        <a href="//localhost:1313/friends/"
           class="">
          Kindred Spirits
        </a>
      
        <a href="//localhost:1313/about/"
           class="">
          About
        </a>
      
    </nav>
  </div>
</header>


    
<main class="content-container">
  <h1>Publications</h1>
  
    
    
            
    
      

      
      
           
        <div class="publication-block">
          <div class="publication-year">2025</div>
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.nature.com/articles/s41467-025-58106-9" target="_blank">
                <img src="//localhost:1313/images//2025_nc_ab.png" alt="Emergence and global spread of asss dominant multidrug-resistant clade within *Acinetobacter baumannii*">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Nature Communications</em></strong></div>
                <div>Li, S., Jiang, G., Wang, S. et al.</div>
                <div><a href="https://doi.org/10.1038/s41467-025-58106-9" target="_blank">10.1038/s41467-025-58106-9</a></div>
                <div class="cite-count" data-doi="10.1038/s41467-025-58106-9">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.nature.com/articles/s41467-025-58106-9" target="_blank">Emergence and global spread of asss dominant multidrug-resistant clade within <em>Acinetobacter baumannii</em></a></h3>

          <div class="abstract">
            <div class="abstract-short">
              The proliferation of multi-drug resistant (MDR) bacteria is driven by the global spread of epidemic lineages that accumulate antimicrobial resistance genes (ARGs). <em>Acinetobacter baumannii</em>, a leading cause of nosocomial infections, displays resistance to most frontline antimicrobials and represents a significant challenge to public health. In this study, we conduct a comprehensive genomic analysis of over 15,000 <em>A. baumannii</em> genomes to …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              The proliferation of multi-drug resistant (MDR) bacteria is driven by the global spread of epidemic lineages that accumulate antimicrobial resistance genes (ARGs). <em>Acinetobacter baumannii</em>, a leading cause of nosocomial infections, displays resistance to most frontline antimicrobials and represents a significant challenge to public health. In this study, we conduct a comprehensive genomic analysis of over 15,000 <em>A. baumannii</em> genomes to identify a predominant epidemic super-lineage (ESL) accounting for approximately 70% of global isolates. Through hierarchical classification of the ESL into distinct lineages, clusters, and clades, we identified a stepwise evolutionary trajectory responsible for the worldwide expansion and transmission of <em>A. baumannii</em> over the last eight decades. We observed the rise and global spread of a previously unrecognized Clade 2.5.6, which emerged in East Asia in 2006. The epidemic of the clade is linked to the ongoing acquisition of ARGs and virulence factors facilitated by genetic recombination. Our results highlight the necessity for One Health-oriented research and interventions to address the spread of this MDR pathogen.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Acinetobacter">Acinetobacter</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Databases">Databases</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Recombination">Recombination</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Metagenomics">Metagenomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Nosocomial&#43;Infection">Nosocomial Infection</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Transmission&#43;Modelling">Transmission Modelling</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://link.springer.com/article/10.1186/s44280-025-00087-x" target="_blank">
                <img src="//localhost:1313/images//2025_oha_pb.png" alt="PathoBase: an integrated One Health approach for zoonotic pathogen genomics analysis">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>One Health Advances</em></strong></div>
                <div>Li, S., Rong, J., Zhong, L., et al.</div>
                <div><a href="https://doi.org/doi.org/10.1186/s44280-025-00087-x" target="_blank">doi.org/10.1186/s44280-025-00087-x</a></div>
                <div class="cite-count" data-doi="doi.org/10.1186/s44280-025-00087-x">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://link.springer.com/article/10.1186/s44280-025-00087-x" target="_blank">PathoBase: an integrated One Health approach for zoonotic pathogen genomics analysis</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              The rapid identification and genomic analysis of zoonotic pathogens are essential for effective disease surveillance and outbreak response. PathoBase, an advanced computational platform developed for researchers, addresses regional challenges in pathogen genomics by offering a comprehensive, efficient, and freely accessible tool for genomic data analysis building on core-genome multilocus sequence typing. The platform currently hosts 132 …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              The rapid identification and genomic analysis of zoonotic pathogens are essential for effective disease surveillance and outbreak response. PathoBase, an advanced computational platform developed for researchers, addresses regional challenges in pathogen genomics by offering a comprehensive, efficient, and freely accessible tool for genomic data analysis building on core-genome multilocus sequence typing. The platform currently hosts 132 bacterial strains, including <em>Acinetobacter</em> (n = 112) and <em>Staphylococcus</em> (n = 20), with additional genera under development. The key objectives of PathoBase include establishing a zoonotic pathogen repository for regional surveillance, developing rapid source-tracing technologies to enhance outbreak control, and promoting interdisciplinary collaboration to strengthen public health security.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Acinetobacter">Acinetobacter</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Databases">Databases</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Staphylococcus">Staphylococcus</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.nature.com/articles/s42003-025-08156-y" target="_blank">
                <img src="//localhost:1313/images//2025_cb_ap.png" alt="Genetic landscape and evolution of *Acinetobacter pittii*, an underestimated emerging nosocomial pathogen">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Communications Biology</em></strong></div>
                <div>Wang, S., Zhou, Y., Wang, Y. et al.</div>
                <div><a href="https://doi.org/10.1038/s42003-025-08156-y" target="_blank">10.1038/s42003-025-08156-y</a></div>
                <div class="cite-count" data-doi="10.1038/s42003-025-08156-y">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.nature.com/articles/s42003-025-08156-y" target="_blank">Genetic landscape and evolution of <em>Acinetobacter pittii</em>, an underestimated emerging nosocomial pathogen</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              As a member of <em>Acinetobacter calcoaceticus-baumannii</em> complex, <em>Acinetobacter pittii</em> has been an emerging concern in nosocomial infection due to its increasing prevalence and multidrug resistance (MDR). However, its population structure remains broadly unknown, hampering efficient tracing of its transmission and evolution. In this study, we developed a distributed core genome multilocus sequence typing (dcgMLST) for <em>A. pittii</em> based on 750 …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              As a member of <em>Acinetobacter calcoaceticus-baumannii</em> complex, <em>Acinetobacter pittii</em> has been an emerging concern in nosocomial infection due to its increasing prevalence and multidrug resistance (MDR). However, its population structure remains broadly unknown, hampering efficient tracing of its transmission and evolution. In this study, we developed a distributed core genome multilocus sequence typing (dcgMLST) for <em>A. pittii</em> based on 750 genomes and employed it to map the genetic landscape and evolution of <em>A. pittii</em>. The results demonstrated that two hierarchical clustering (HC) levels effectively correspond to genetic diversity from species (HC1100) to natural populations (HC450), as well as that a predominant lineage, HC1100_4, accounts for 33.9% of <em>A. pittii</em> strains. Subsequent analysis revealed that specific gene gain and loss events within HC1100_4 are linked to adaptations to environmental stress. Moreover, we identified a cluster of multidrug-resistant plasmids PT_712 responsible for the dissemination of <em>bla</em>NDM-1 genes within the genus of <em>Acinetobacter</em>. This study provides a framework for characterizing genetic diversity, evolutionary dynamics, molecular population distribution, and tracing of <em>A. pittii</em>, which has the potential to improve infection control strategies and public health policy.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Acinetobacter">Acinetobacter</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Databases">Databases</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Plasmids">Plasmids</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Nosocomial&#43;Infection">Nosocomial Infection</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Transmission&#43;Modelling">Transmission Modelling</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.sciencedirect.com/science/article/abs/pii/S0048969724084651" target="_blank">
                <img src="//localhost:1313/images//2025_stoten_ss.png" alt="Uncovering a cryptic *Streptococcus suis* endemic post-outbreak: Evidence of host switching to humans">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Science of The Total Environment</em></strong></div>
                <div>Zhu, S., Li, S., Wu, B. et al.</div>
                <div><a href="https://doi.org/10.1016/j.scitotenv.2024.178307" target="_blank">10.1016/j.scitotenv.2024.178307</a></div>
                <div class="cite-count" data-doi="10.1016/j.scitotenv.2024.178307">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.sciencedirect.com/science/article/abs/pii/S0048969724084651" target="_blank">Uncovering a cryptic <em>Streptococcus suis</em> endemic post-outbreak: Evidence of host switching to humans</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              <em>Streptococcus suis</em> (<em>S. suis</em>) is a neglected and emerging pathogen that leads to severe economic losses in swine industry. Despite its epidemic potential, the zoonotic threat posed by <em>S. suis</em> remains underappreciated, even after the unprecedented Sichuan outbreak, which highlighted its ability to cause fatal human infections. Understanding of the dynamics and evolution of this pathogen in human populations is crucial for preventing future …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              <em>Streptococcus suis</em> (<em>S. suis</em>) is a neglected and emerging pathogen that leads to severe economic losses in swine industry. Despite its epidemic potential, the zoonotic threat posed by <em>S. suis</em> remains underappreciated, even after the unprecedented Sichuan outbreak, which highlighted its ability to cause fatal human infections. Understanding of the dynamics and evolution of this pathogen in human populations is crucial for preventing future outbreaks. Our study revealed the emergence of highly pathogenic <em>S. suis</em> lineages in Zhejiang Province following the Sichuan outbreak, showing an increasingly specialized lifestyle that has persisted for nearly two decades. Phylogenetic analysis traced the zoonotic transmission of this pathogen back to a livestock lineage in the Netherlands prior to 1990, which eventually led to the Sichuan outbreak lineage in 2005 and its subsequent spread to Zhejiang the same year. Two independent evolved sub-lineages were identified in Zhejiang, suggesting a cryptic, regional endemicity following the Sichuan outbreak. Furthermore, the accumulation of lineage-specific resistance and metabolic acclimation after divergence from the Sichuan population suggested potential regional evolutionary shifts in <em>S. suis</em>. These new findings could help inform future intervention strategies and guide public health policies.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Streptococcus">Streptococcus</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Metabolite&#43;Reconstruction">Metabolite Reconstruction</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Human-Animals&#43;Infection">Human-Animals Infection</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Transmission&#43;Modelling">Transmission Modelling</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=5043757" target="_blank">
                <img src="//localhost:1313/images//2025_pp_ab.png" alt="Multicenter Genomic Epidemiology Reveals Distinct Short-Term and Long-Term Crab Transmission Patterns in Zhejiang, China">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Preprint</em></strong></div>
                <div>Li, S., Wang, S., Pu, F. et al.</div>
                <div><a href="https://doi.org/10.2139/ssrn.5043757" target="_blank">10.2139/ssrn.5043757</a></div>
                <div class="cite-count" data-doi="10.2139/ssrn.5043757">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=5043757" target="_blank">Multicenter Genomic Epidemiology Reveals Distinct Short-Term and Long-Term Crab Transmission Patterns in Zhejiang, China</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              <strong>Objectives</strong>  To investigate the genomic epidemiology of carbapenem-resistant <em>Acinetobacter baumannii</em> (CRAB) in ICUs of municipal hospitals in Zhejiang province, and to elucidate its transmission patterns and determine its spread influencing factors.   <strong>Methods</strong> We conducted a multicenter genomic epidemiological analysis of 138 CRAB isolates collected from ICUs of tertiary hospitals across 8 prefecture-level cities in Zhejiang province …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              <strong>Objectives</strong>  To investigate the genomic epidemiology of carbapenem-resistant <em>Acinetobacter baumannii</em> (CRAB) in ICUs of municipal hospitals in Zhejiang province, and to elucidate its transmission patterns and determine its spread influencing factors.   <strong>Methods</strong> We conducted a multicenter genomic epidemiological analysis of 138 CRAB isolates collected from ICUs of tertiary hospitals across 8 prefecture-level cities in Zhejiang province between 2018 and 2023. Antimicrobial susceptibility testing and whole 49 genome sequencing were performed on these isolates. By integrating genomic data from the capital city in public database, we identified population structures and reconstructed transmission events to assess dynamic changes over time.    <strong>Results</strong> Between 2018 and 2023, CRAB isolates from tertiary hospitals in 9 prefecture-level cities in Zhejiang Province predominantly consisted of the ESL2.4 and ESL2.5 lineages. The profile of antimicrobial resistance genes was strongly associated with phylogenetic lineage. Short-term outbreaks identified using a 5-SNP threshold, demonstrated a significant correlation with geographical distances within 160 km. Temporal phylogenetic analysis revealed that the prolonged and sustained transmission of CRAB strains across ICUs in Zhejiang originating in the 2010s, coincided notably with the rapid development and enhancement of traffic and public transportation in the provincial capital starting in 2010.  <strong>Conclusions</strong> CRAB in ICUs from 9 hospitals in Zhejiang Province predominantly consisted of the ESL2.4 and ESL2.5 lineages. Geographical distance and transportation are critical factors influencing the transmission of CRAB outbreaks among ICUs in different cities. The provincial capital exerts a persistent and significant influence on non-capital cities beyond geographical constraints.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Acinetobacter">Acinetobacter</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Plasmids">Plasmids</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Transmission&#43;Modelling">Transmission Modelling</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Nosocomial&#43;Infection">Nosocomial Infection</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.sciencedirect.com/science/article/abs/pii/S092485792500144X" target="_blank">
                <img src="//localhost:1313/images//2025_ijaa_se.jpg" alt="Emergence of carbapenem-resistant XDR *Salmonella enterica* in pediatric patients in South China: a genomic perspective study">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>International Journal of Antimicrobial Agents</em></strong></div>
                <div>Feng, X., Li, S., Huang, D. et al.</div>
                <div><a href="https://doi.org/10.1016/j.ijantimicag.2025.107589" target="_blank">10.1016/j.ijantimicag.2025.107589</a></div>
                <div class="cite-count" data-doi="10.1016/j.ijantimicag.2025.107589">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.sciencedirect.com/science/article/abs/pii/S092485792500144X" target="_blank">Emergence of carbapenem-resistant XDR <em>Salmonella enterica</em> in pediatric patients in South China: a genomic perspective study</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              <strong>Background and Aim</strong>  Carbapenem-resistant <em>Salmonella enterica</em> (CRSE), mostly driven by plasmids, poses a growing public health threat, especially in paediatric populations. This study investigates a cluster of paediatric CRSE infections in paediatric populations, characterizes genomic features of CRSE isolates, assesses global CRSE prevalence, and explores plasmid-mediated horizontal gene transfer.  <strong>Methods</strong>  An epidemiological …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              <strong>Background and Aim</strong>  Carbapenem-resistant <em>Salmonella enterica</em> (CRSE), mostly driven by plasmids, poses a growing public health threat, especially in paediatric populations. This study investigates a cluster of paediatric CRSE infections in paediatric populations, characterizes genomic features of CRSE isolates, assesses global CRSE prevalence, and explores plasmid-mediated horizontal gene transfer.  <strong>Methods</strong>  An epidemiological investigation of 18 paediatric CRSE cases was conducted. Genomic analysis included resistome profiling, plasmid typing, and phylogenetic clustering to assess genetic diversity. A global analysis of 530,113 <em>Salmonella genomes</em> identified carbapenemase-carrying isolates. Plasmid transfer experiments between <em>S. enterica</em> and <em>Escherichia coli</em> were performed to evaluate horizontal gene transmission.  <strong>Results</strong>  Respiratory co-infections (67% of cases, primarily respiratory syncytial virus and human parainfluenza viruses) were associated with severe clinical outcomes. Genomic analysis revealed multiple genetically distinct CRSE clones carrying blaNDM-5, predominantly on IncI-gamma/K1 and IncHI2A plasmids. Plasmid-mediated transfer of carbapenem resistance genes between <em>S. enterica</em> and <em>E. coli</em> was confirmed. Global surveillance identified 228 carbapenemase-positive Salmonella isolates (2000–2023) across 35 genetically diverse populations and 24 countries, demonstrating widespread dissemination.  <strong>Conclusions</strong>  Respiratory co-infections may exacerbate CRSE severity in children, while plasmid circulation drives carbapenem resistance transmission. The high genetic diversity and global distribution of CRSE highlight urgent needs for integrated surveillance, antimicrobial stewardship, and interventions targeting co-infections and environmental reservoirs.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Transmission&#43;Modelling">Transmission Modelling</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Salmonella">Salmonella</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Nosocomial&#43;Infection">Nosocomial Infection</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="//localhost:1313/NA" target="_blank">
                <img src="//localhost:1313/images//2025_emi_mp.png" alt="Post-COVID-19 disruption of the respiratory microbiome modulates *Mycoplasma* pneumoniae: a multi-center retrospective investigation study">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Emerging Microbes &amp; Infections</em></strong></div>
                <div>Huang, S., Li, S., Zhao, R. et al.</div>
                <div><a href="https://doi.org/Accepted" target="_blank">Accepted</a></div>
                <div class="cite-count" data-doi="Accepted">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="NA" target="_blank">Post-COVID-19 disruption of the respiratory microbiome modulates <em>Mycoplasma</em> pneumoniae: a multi-center retrospective investigation study</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              Since the COVID-19 pandemic, there has been a notable resurgence of Mycoplasma pneumoniae pneumonia (MPP) in children, with a concerning rise in the severity of cases. Although changes in post-pandemic respiratory infection patterns have been documented, the reasons behind the increased severity of MPP, especially concerning shifts in the respiratory microbiota, are not well understood. This study aims to explore how pandemic-associated …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              Since the COVID-19 pandemic, there has been a notable resurgence of Mycoplasma pneumoniae pneumonia (MPP) in children, with a concerning rise in the severity of cases. Although changes in post-pandemic respiratory infection patterns have been documented, the reasons behind the increased severity of MPP, especially concerning shifts in the respiratory microbiota, are not well understood. This study aims to explore how pandemic-associated disruptions in respiratory microbiota contribute to MPP severity. Through analysis of multiple independent cohorts, we found that the depletion of protective respiratory microbiota exacerbates MPP severity by reducing colonization resistance again135921@pyzYAst M. pneumoniae. We identified two antagonistic microbiota modules with distinct metabolic features and community structures that regulate pathogen colonization and disease severity. Using a Susceptible-Infected-Recovered (SIR) model, we simulated M. pneumoniae infection dynamics across different microbiota states. Additionally, a machine-learning model based on eight key microbes effectively distinguished between cases and controls and predicted infection severity, offering insights into post-pandemic respiratory infections. This study underscores the essential role of the respiratory microbiota in influencing the severity of M. pneumoniae infections. Our findings offer a framework for managing respiratory infections in the post-pandemic era, highlighting the significance of understanding microbial community dynamics in determining disease outcomes. These insights could guide the development of therapeutic strategies focused on restoring or enhancing the protective microbiota, which may offer an effective strategy for the management of MPP and other respiratory infections.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Metagenomics">Metagenomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Mycoplasma">Mycoplasma</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Community&#43;Infection">Community Infection</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Community&#43;Modelling">Community Modelling</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Microbiome">Microbiome</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/imt2.70077" target="_blank">
                <img src="//localhost:1313/images//2025_imeta_kp.jpg" alt="Geographic containment and virulence-resistance trade-offs drive the evolution of hypervirulent *Klebsiella pneumoniae*">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>iMeta</em></strong></div>
                <div>Wu, Y., Pu, F., Yan, Z. et al.</div>
                <div><a href="https://doi.org/doi.org/10.1002/imt2.70077" target="_blank">doi.org/10.1002/imt2.70077</a></div>
                <div class="cite-count" data-doi="doi.org/10.1002/imt2.70077">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/imt2.70077" target="_blank">Geographic containment and virulence-resistance trade-offs drive the evolution of hypervirulent <em>Klebsiella pneumoniae</em></a></h3>

          <div class="abstract">
            <div class="abstract-short">
              The emergence of hypervirulent carbapenem-resistant <em>Klebsiella pneumoniae</em> (hvCRKP) represents an alarming convergence of enhanced virulence and extensive drug resistance. Here, we present a comprehensive genomic analysis of 2563 clonal complex 23 (CC23) isolates from 62 countries spanning 1932–2024. Our findings reveal that CC23-K1, the dominant hypervirulent sublineage, emerged approximately 170 years ago and diversified into seven major …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              The emergence of hypervirulent carbapenem-resistant <em>Klebsiella pneumoniae</em> (hvCRKP) represents an alarming convergence of enhanced virulence and extensive drug resistance. Here, we present a comprehensive genomic analysis of 2563 clonal complex 23 (CC23) isolates from 62 countries spanning 1932–2024. Our findings reveal that CC23-K1, the dominant hypervirulent sublineage, emerged approximately 170 years ago and diversified into seven major clades with distinct regional dominance. We observe that carbapenem resistance in CC23-K1 exhibits notable instability, with at least 130 independent acquisitions and 20 losses of resistance genes, suggesting an evolutionary trade-off between hypervirulence and antimicrobial resistance. Experimental validation demonstrates that capsule production physically impedes plasmid conjugation, while isolates carrying blaKPC-2, blaNDM-1, or blaNDM-5 frequently exhibit substantial deletion of virulence determinants. Conversely, blaOXA-48-carrying isolates maintain virulence gene integrity, potentially due to their lower hydrolytic activity and reduced fitness costs. The geographic distribution of these resistance mechanisms correlates with regional antimicrobial usage patterns, with European countries with moderate carbapenem use favoring blaOXA-48 in CC23, while Asian countries with higher consumption show patterns favoring high-efficiency carbapenemases incompatible with complete virulence determinants. We also identified core genomic regions with significantly higher mutation rates in resistant isolates, particularly affecting pathways involved in oxidative phosphorylation and reactive oxygen species production. These findings provide additional insights into CC23 evolution and geographical spread, complementing existing knowledge of carbapenemase distribution patterns observed across <em>K. pneumoniae</em> lineages.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Klebsiella">Klebsiella</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Transmission&#43;Modelling">Transmission Modelling</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Nosocomial&#43;Infection">Nosocomial Infection</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      
        </div>   
        <div class="publication-block">
          <div class="publication-year">2024</div>
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://link.springer.com/article/10.1186/s13073-024-01399-0" target="_blank">
                <img src="//localhost:1313/images//2024_gm_klety.png" alt="KleTy: integrated typing scheme for core genome and plasmids reveals repeated emergence of multi-drug resistant epidemic lineages in *Klebsiella* worldwide">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Genome Medicine</em></strong></div>
                <div>Li, H., Liu, X., Li, S. et al.</div>
                <div><a href="https://doi.org/10.1186/s13073-024-01399-0" target="_blank">10.1186/s13073-024-01399-0</a></div>
                <div class="cite-count" data-doi="10.1186/s13073-024-01399-0">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://link.springer.com/article/10.1186/s13073-024-01399-0" target="_blank">KleTy: integrated typing scheme for core genome and plasmids reveals repeated emergence of multi-drug resistant epidemic lineages in <em>Klebsiella</em> worldwide</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              <strong>Background</strong>   Clinically important lineages in <em>Klebsiella</em>, especially those expressing multi-drug resistance (MDR), pose severe threats to public health worldwide. They arose from the co-evolution of the vertically inherited core genome and horizontal gene transfers by plasmids, which has not been systematically explored.   <strong>Methods</strong> We designed KleTy, which consists of dedicated typing schemes for both the core genome and plasmids in …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              <strong>Background</strong>   Clinically important lineages in <em>Klebsiella</em>, especially those expressing multi-drug resistance (MDR), pose severe threats to public health worldwide. They arose from the co-evolution of the vertically inherited core genome and horizontal gene transfers by plasmids, which has not been systematically explored.   <strong>Methods</strong> We designed KleTy, which consists of dedicated typing schemes for both the core genome and plasmids in <em>Klebsiella</em>. We compared the performance of KleTy with many state-of-the-art pipelines using both simulated and real data.    <strong>Results</strong> Employing KleTy, we genotyped 33,272 <em>Klebsiella</em> genomes, categorizing them into 1773 distinct populations and predicting the presence of 87,410 plasmids from 837 clusters (PCs). Notably, <em>Klebsiella</em> is the center of the plasmid-exchange network within <em>Enterobacteriaceae</em>. Our results associated the international emergence of prevalent <em>Klebsiella</em> populations with only four carbapenem-resistance (CR) PCs, two hypervirulent PCs, and two hvCR-PCs encoding both carbapenemase and hypervirulence. Furthermore, we observed the ongoing international emergence of <em>bla</em>NDM, accompanied by the replacement of the previously dominant population, <em>bla</em>KPC-encoding HC1360_8 (CC258), during 2003–2018, with the emerging <em>bla</em>NDM-encoding HC1360_3 (CC147) thereafter. Additionally, expansions of hypervirulent carbapenem-resistant <em>Klebsiella pneumoniae</em> (hvCRKP) were evidenced in both populations, driven by plasmids of MDR-hypervirulence convergences.  <strong>Conclusions</strong> The study illuminates how the global genetic landscape of <em>Klebsiella</em> has been shaped by the co-evolution of both the core genome and the plasmids, underscoring the importance of surveillance and control of the dissemination of plasmids for curtailing the emergence of hvCRKPs.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Klebsiella">Klebsiella</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Databases">Databases</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Plasmids">Plasmids</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Nosocomial&#43;Infection">Nosocomial Infection</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.sciencedirect.com/science/article/pii/S1201971224001255" target="_blank">
                <img src="//localhost:1313/images//2024_ijid_cj.jpg" alt="Epidemiological and molecular investigations of sequential outbreaks of *Campylobacter jejuni* infecting adults and schoolchildren in southeastern China, 2021-2022">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>International Journal of Infectious Diseases</em></strong></div>
                <div>Zhang, L., Shao, Y., Li, S. et al.</div>
                <div><a href="https://doi.org/10.1016/j.ijid.2024.107055" target="_blank">10.1016/j.ijid.2024.107055</a></div>
                <div class="cite-count" data-doi="10.1016/j.ijid.2024.107055">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.sciencedirect.com/science/article/pii/S1201971224001255" target="_blank">Epidemiological and molecular investigations of sequential outbreaks of <em>Campylobacter jejuni</em> infecting adults and schoolchildren in southeastern China, 2021-2022</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              <strong>Objectives</strong>  To investigate cases of five <em>Campylobacter jejuni</em> outbreaks and describe laboratory characteristics of these infections.   <strong>Methods</strong> Whole-genome sequencing and conventional methods were combined to thoroughly investigate the outbreaks, and data of contemporaneous sporadic cases was included for comparison.    <strong>Results</strong> Seven sequence types (STs) of <em>C. jejuni</em> caused 83 cases, including ST9079 which recurred across 2 years. …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              <strong>Objectives</strong>  To investigate cases of five <em>Campylobacter jejuni</em> outbreaks and describe laboratory characteristics of these infections.   <strong>Methods</strong> Whole-genome sequencing and conventional methods were combined to thoroughly investigate the outbreaks, and data of contemporaneous sporadic cases was included for comparison.    <strong>Results</strong> Seven sequence types (STs) of <em>C. jejuni</em> caused 83 cases, including ST9079 which recurred across 2 years. Trace-back investigation could not identify any food items of infection but detected identical campylobacters from food contacts. Phylogenetic analysis unveiled genetic closeness between outbreak strains and some concurrent sporadic strains, indicating local campylobacteriosis may not be wholly sporadic but rather a series of linked cases. Virulence genes disclosed species/case-specific signatures to differentiate outbreaks from truly non-outbreak strains. Resistance to fluoroquinolones and/or macrolides was prevalent (90.8%, 108/119), with a noteworthy portion exhibiting multidrug resistance (31.1%, 37/119). Five types of plasmids were harbored among outbreak isolates, of which one plasmid harboring anti-stress and resistant genes was rarely found in <em>C. jejuni</em>.  <strong>Conclusions</strong> This is the first reported sequential outbreak of <em>C. jejuni</em> in China. Our observations help to define the genomic landscape and antimicrobial resistance patterns of <em>Campylobacter</em>, emphasizing the need for a broader ‘One Health’ perspective to combat the threats posed by campylobacteriosis.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Campylobacter">Campylobacter</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Food-borne&#43;Infection">Food-borne Infection</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Community&#43;Infection">Community Infection</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247%2823%2900405-6/fulltext" target="_blank">
                <img src="//localhost:1313/images//2024_lm_mp.png" alt="Resurgence of *Mycoplasma* pneumonia by macrolide-resistant epidemic clones in China">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>The Lancet Microbe</em></strong></div>
                <div>Li, H., Li, S., Yang, H. et al.</div>
                <div><a href="https://doi.org/10.1016/S2666-5247%2823%2900405-6" target="_blank">10.1016/S2666-5247(23)00405-6</a></div>
                <div class="cite-count" data-doi="10.1016/S2666-5247(23)00405-6">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247%2823%2900405-6/fulltext" target="_blank">Resurgence of <em>Mycoplasma</em> pneumonia by macrolide-resistant epidemic clones in China</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              We found that EC2 showed 100% macrolide resistance because of the acquisition of the A2063G mutation in 23S rRNA. All three P1-2 strains isolated in Beijing in 2020 were derived from EC2, suggesting that EC2 could be responsible for the previously reported increase in MRMP frequencies in P1-2 strains detected in the country after 2018. Moreover, our analyses showed frequent, cryptic cross-regional <em>M pneumoniae</em> transmissions in both EC1 and EC2, …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              We found that EC2 showed 100% macrolide resistance because of the acquisition of the A2063G mutation in 23S rRNA. All three P1-2 strains isolated in Beijing in 2020 were derived from EC2, suggesting that EC2 could be responsible for the previously reported increase in MRMP frequencies in P1-2 strains detected in the country after 2018. Moreover, our analyses showed frequent, cryptic cross-regional <em>M pneumoniae</em> transmissions in both EC1 and EC2, and the Taiwan strains in 2019 showed that the EC2 MRMP strain had been spreading throughout China before 2020. PCR-based surveillance data also suggested an unusual increase in infections due to the P1-2 MRMP strain in north China from 2021 to 2022. Our findings suggest that without the restrictions enacted during the COVID-19 pandemic, the MRMP clones could have caused outbreaks across the country as early as 2020; thus, our findings provide evidence to guide future epidemic prevention and treatment in east Asia.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Metagenomics">Metagenomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Mycoplasma">Mycoplasma</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Community&#43;Infection">Community Infection</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.nature.com/articles/s43016-024-00968-1" target="_blank">
                <img src="//localhost:1313/images//2024_nf_se.png" alt="Centralized industrialization of pork in Europe and America contributes to the global spread of *Salmonella enterica*">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Nature Food</em></strong></div>
                <div>Li, H., Wu, Y., Feng, D. et al.</div>
                <div><a href="https://doi.org/10.1038/s43016-024-00968-1" target="_blank">10.1038/s43016-024-00968-1</a></div>
                <div class="cite-count" data-doi="10.1038/s43016-024-00968-1">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.nature.com/articles/s43016-024-00968-1" target="_blank">Centralized industrialization of pork in Europe and America contributes to the global spread of <em>Salmonella enterica</em></a></h3>

          <div class="abstract">
            <div class="abstract-short">
              <em>Salmonella enterica</em> causes severe food-borne infections through contamination of the food supply chain. Its evolution has been associated with human activities, especially animal husbandry. Advances in intensive farming and global transportation have substantially reshaped the pig industry, but their impact on the evolution of associated zoonotic pathogens such as <em>S. enterica</em> remains unresolved. Here we investigated the population …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              <em>Salmonella enterica</em> causes severe food-borne infections through contamination of the food supply chain. Its evolution has been associated with human activities, especially animal husbandry. Advances in intensive farming and global transportation have substantially reshaped the pig industry, but their impact on the evolution of associated zoonotic pathogens such as <em>S. enterica</em> remains unresolved. Here we investigated the population fluctuation, accumulation of antimicrobial resistance genes and international serovar Choleraesuis transmission of nine pig-enriched <em>S. enterica</em> populations comprising more than 9,000 genomes. Most changes were found to be attributable to the developments of the modern pig industry. All pig-enriched salmonellae experienced host transfers in pigs and/or population expansions over the past century, with pigs and pork having become the main sources of <em>S. enterica</em> transmissions to other hosts. Overall, our analysis revealed strong associations between the transmission of pig-enriched salmonellae and the global pork trade.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Transmission&#43;Modelling">Transmission Modelling</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Salmonella">Salmonella</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Bacteria&#43;genetics">Bacteria genetics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Genomics">Genomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Nosocomial&#43;Infection">Nosocomial Infection</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      
        </div>   
        <div class="publication-block">
          <div class="publication-year">2023</div>
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1169476/full" target="_blank">
                <img src="//localhost:1313/images//2023_fm_wzwater.png" alt="Characterization of microbial community and antibiotic resistome in intra urban water, Wenzhou China">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Frontiers in Microbiology</em></strong></div>
                <div>Ye, S., Li, S., Su, C. et al.</div>
                <div><a href="https://doi.org/10.3389/fmicb.2023.1169476" target="_blank">10.3389/fmicb.2023.1169476</a></div>
                <div class="cite-count" data-doi="10.3389/fmicb.2023.1169476">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1169476/full" target="_blank">Characterization of microbial community and antibiotic resistome in intra urban water, Wenzhou China</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              The present study investigated the water quality index, microbial composition and antimicrobial resistance genes in urban water habitats. Combined chemicals testing, metagenomic analyses and qualitative PCR (qPCR) were conducted on 20 locations, including rivers from hospital surrounds (n = 7), community surrounds (n = 7), and natural wetlands (n = 6). Results showed that the indexes of total nitrogen, phosphorus, and ammonia nitrogen of hospital …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              The present study investigated the water quality index, microbial composition and antimicrobial resistance genes in urban water habitats. Combined chemicals testing, metagenomic analyses and qualitative PCR (qPCR) were conducted on 20 locations, including rivers from hospital surrounds (n = 7), community surrounds (n = 7), and natural wetlands (n = 6). Results showed that the indexes of total nitrogen, phosphorus, and ammonia nitrogen of hospital waters were 2–3 folds high than that of water from wetlands. Bioinformatics analysis revealed a total of 1,594 bacterial species from 479 genera from the three groups of water samples. The hospital-related samples had the greatest number of unique genera, followed by those from wetlands and communities. The hospital-related samples contained a large number of bacteria associated with the gut microbiome, including <em>Alistipes</em>, <em>Prevotella</em>, <em>Klebsiella</em>, <em>Escherichia</em>, <em>Bacteroides</em>, and <em>Faecalibacterium</em>, which were all significantly enriched compared to samples from the wetlands. Nevertheless, the wetland waters enriched bacteria from <em>Nanopelagicus</em>, <em>Mycolicibacterium</em> and <em>Gemmatimonas</em>, which are typically associated with aquatic environments. The presence of antimicrobial resistance genes (ARGs) that were associated with different species origins in each water sample was observed. The majority of ARGs from hospital-related samples were carried by bacteria from <em>Acinetobacter</em>, <em>Aeromonas</em> and various genera from <em>Enterobacteriaceae</em>, which each was associated with multiple ARGs. In contrast, the ARGs that were exclusively in samples from communities and wetlands were carried by species that encoded only 1 to 2 ARGs each and were not normally associated with human infections. The qPCR showed that water samples of hospital surrounds had higher concentrations of intI1 and antimicrobial resistance genes such as <em>tetA</em>, <em>ermA</em>, <em>ermB</em>, <em>qnrB</em>, <em>sul1</em>, <em>sul2</em> and other beta-lactam genes. Further genes of functional metabolism reported that the enrichment of genes associated with the degradation/utilization of nitrate and organic phosphodiester were detected in water samples around hospitals and communities compared to those from wetlands. Finally, correlations between the water quality indicators and the number of ARGs were evaluated. The presence of total nitrogen, phosphorus, and ammonia nitrogen were significantly correlated with the presence of <em>ermA</em> and <em>sul1</em>. Furthermore, intI1 exhibited a significant correlation with <em>ermB</em>, <em>sul1</em>, and <em>bla</em>SHV, indicating a prevalence of ARGs in urban water environments might be due to the integron intI1’s diffusion-promoting effect. However, the high abundance of ARGs was limited to the waters around the hospital, and we did not observe the geographical transfer of ARGs along with the river flow. This may be related to water purifying capacity of natural riverine wetlands. Taken together, continued surveillance is required to assess the risk of bacterial horizontal transmission and its potential impact on public health in the current region.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=AMR">AMR</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Metagenomics">Metagenomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Microbiome">Microbiome</a>
            
          </div>
          
        </div>
      </div>

        
    
      

      
      

      <div class="publication-entry">
        
        <div class="publication-image-meta">
          
            <div class="publication-image-wrapper">
              <a href="https://www.frontiersin.org/journals/cellular-and-infection-microbiology/articles/10.3389/fcimb.2022.1090889/full" target="_blank">
                <img src="//localhost:1313/images//2023_fcim_ptb.png" alt="The correlation between dysfunctional intestinal flora and pathology feature of patients with pulmonary tuberculosis">
              </a>
              <div class="publication-meta-card">
                <div><strong><em>Frontiers in Cellular and Infection Microbiology</em></strong></div>
                <div>Ye, S., Wang, L., Li, S. et al.</div>
                <div><a href="https://doi.org/10.3389/fcimb.2022.1090889" target="_blank">10.3389/fcimb.2022.1090889</a></div>
                <div class="cite-count" data-doi="10.3389/fcimb.2022.1090889">Citations: …</div>
              </div>
            </div>
          
        </div>

        
        <div class="publication-text" style="">
          <h3><a href="https://www.frontiersin.org/journals/cellular-and-infection-microbiology/articles/10.3389/fcimb.2022.1090889/full" target="_blank">The correlation between dysfunctional intestinal flora and pathology feature of patients with pulmonary tuberculosis</a></h3>

          <div class="abstract">
            <div class="abstract-short">
              Recent studies have provided insights into the important contribution of gut microbiota in the development of Pulmonary Tuberculosis (PTB). As a chronic consumptive infectious disease, PTB involves many pathological characteristics. At present, research on intestinal flora and clinical pathological Index of PTB is still rare. We performed a cross-sectional study in 63 healthy controls (HCs) and 69 patients with untreated active PTB to assess the …
              <span class="read-more-toggle">[Read more]</span>
            </div>
            <div class="abstract-full hidden">
              Recent studies have provided insights into the important contribution of gut microbiota in the development of Pulmonary Tuberculosis (PTB). As a chronic consumptive infectious disease, PTB involves many pathological characteristics. At present, research on intestinal flora and clinical pathological Index of PTB is still rare. We performed a cross-sectional study in 63 healthy controls (HCs) and 69 patients with untreated active PTB to assess the differences in their microbiota in feces via 16S rRNA gene sequencing. Significant alteration of microbial taxonomic and functional capacity was observed in PTB as compared to the HCs. The results showed that the alpha diversity indexes of the PTB patients were lower than the HCs (P&lt;0.05). Beta diversity showed differences between the two groups (P&lt;0.05). At the genus level, the relative abundance of Bacteroides, Parabacteroides and Veillonella increased, while Faecalibacterium, Bifidobacterium, Agathobacter and CAG-352 decreased significantly in the PTB group, when compared with the HCs. The six combined genera, including <em>Lactobacillus</em>, <em>Faecalibacterium</em>, <em>Roseburia</em>, <em>Dorea</em>, <em>Monnoglobus</em> and [Eubacterium]_ventriosum_group might be a set of diagnostic biomarkers for PTB (AUC=0.90). Besides, the predicted bacterial functional pathway had a significant difference between the two groups (P&lt;0.05), which was mainly related to the nutrient metabolism pathway. Significant alterations in the biochemical index were associated with changes in the relative abundance of specific bacteria, the short chain fatty acid (SCFA)-producing bacteria enriched in HCs had a positively correlated with most of the biochemical indexes. Our study indicated that the gut microbiota in PTB patients was significantly different from HCs as characterized by the composition and metabolic pathway, which related to the change of biochemical indexes in the PTB group. It was hypothesized that the abovementioned changes in the gut microbiota could exert an impact on the clinical characteristics of PTB through the regulation of the nutrient utilization pathway of the host by way of the gut-lung axis.
              <span class="read-more-toggle">[Read less]</span>
            </div>
          </div>

          
          <div class="pub-keywords">
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Metagenomics">Metagenomics</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Microbiome">Microbiome</a>
            
              <a class="keyword-badge" href="//localhost:1313/arts/?keyword=Mycobacteria">Mycobacteria</a>
            
          </div>
          
        </div>
      </div>

        
    
  
  </div>                       
</main>

<style>
.pub-keywords {
  margin-top: 12px;
  display: flex;
  flex-wrap: wrap;
  gap: 8px;
}
.keyword-badge {
  display: inline-block;
  padding: 4px 12px;
  border-radius: 20px;
  background: #f6f6f6;
  box-shadow: 2px 2px 4px rgba(0,0,0,0.15);
  font-size: 13px;
  color: #444;
  text-decoration: none;
  transition: all 0.2s;
}
.keyword-badge:hover {
  background-color: #ffe4e1;
  transform: translateY(-1px);
  color: #b33c3c;
}
</style>

<script>
document.addEventListener('DOMContentLoaded', () => {
  const spans = [...document.querySelectorAll('.cite-count')];

  
  const cache = new Map();

  spans.forEach(span => {
    const doi = span.dataset.doi;
    if (!doi) return;

    
    if (cache.has(doi)) {
      span.textContent = `Citations: ${cache.get(doi)}`;
      return;
    }

    
    const url = `https://api.openalex.org/works/doi:${encodeURIComponent(doi.toLowerCase())}`;

    fetch(url)
      .then(r => r.ok ? r.json() : Promise.reject(r))
      .then(data => {
        const count = data.cited_by_count ?? 0;
        cache.set(doi, count);
        span.textContent = `Citations: ${count}`;
      })
      .catch(() => {
        span.textContent = 'Citations: n/a';
      });
  });
});
</script>


    <footer class="site-footer" style="text-align: center; padding: 2rem 0;">
  <div class="social-icons">
    <a href="https://github.com/naclist" target="_blank">
      <img class="social-icon" src="//localhost:1313/images/github-color.svg" alt="GitHub">
    </a>
    <a href="https://scholar.google.com/citations?user=jjvdIuYAAAAJ" target="_blank">
      <img class="social-icon" src="//localhost:1313/images/scholar-color.svg" alt="Google Scholar">
    </a>
    <a href="mailto:Zxzzzhu@gmail.com">
      <img class="social-icon" src="//localhost:1313/images/email-color.svg" alt="Email">
    </a>
  </div>
  <p>© 2025 Naclist. All rights reserved.</p>
</footer>

    <script>
document.addEventListener("DOMContentLoaded", () => {
  document.addEventListener("click", e => {
    if (!e.target.classList.contains("read-more-toggle")) return;
    const box = e.target.closest(".abstract");
    box.querySelector(".abstract-short").classList.toggle("hidden");
    box.querySelector(".abstract-full").classList.toggle("hidden");
  });
});
</script>
  </body>
</html>
