---
title: "Publications"

papers:
  - title: "Emergence and global spread of a dominant multidrug-resistant clade within *Acinetobacter baumannii*"
    abstract: >
      The proliferation of multi-drug resistant (MDR) bacteria is driven by the global spread of epidemic lineages that accumulate antimicrobial resistance genes (ARGs). *Acinetobacter baumannii*, a leading cause of nosocomial infections, displays resistance to most frontline antimicrobials and represents a significant challenge to public health. In this study, we conduct a comprehensive genomic analysis of over 15,000 *A. baumannii* genomes to identify a predominant epidemic super-lineage (ESL) accounting for approximately 70% of global isolates. Through hierarchical classification of the ESL into distinct lineages, clusters, and clades, we identified a stepwise evolutionary trajectory responsible for the worldwide expansion and transmission of *A. baumannii* over the last eight decades. We observed the rise and global spread of a previously unrecognized Clade 2.5.6, which emerged in East Asia in 2006. The epidemic of the clade is linked to the ongoing acquisition of ARGs and virulence factors facilitated by genetic recombination. Our results highlight the necessity for One Health-oriented research and interventions to address the spread of this MDR pathogen.
    year: 2025
    image: "2025_nc_ab.png"
    journal: "Nature Communications"
    authors: "Li, S., Jiang, G., Wang, S. et al."
    doi: "10.1038/s41467-025-58106-9"
    link: "https://www.nature.com/articles/s41467-025-58106-9"

  - title: "Genetic landscape and evolution of *Acinetobacter pittii*, an underestimated emerging nosocomial pathogen"
    abstract: >
      As a member of *Acinetobacter calcoaceticus-baumannii* complex, *Acinetobacter pittii* has been an emerging concern in nosocomial infection due to its increasing prevalence and multidrug resistance (MDR). However, its population structure remains broadly unknown, hampering efficient tracing of its transmission and evolution. In this study, we developed a distributed core genome multilocus sequence typing (dcgMLST) for *A. pittii* based on 750 genomes and employed it to map the genetic landscape and evolution of *A. pittii*. The results demonstrated that two hierarchical clustering (HC) levels effectively correspond to genetic diversity from species (HC1100) to natural populations (HC450), as well as that a predominant lineage, HC1100_4, accounts for 33.9% of *A. pittii* strains. Subsequent analysis revealed that specific gene gain and loss events within HC1100_4 are linked to adaptations to environmental stress. Moreover, we identified a cluster of multidrug-resistant plasmids PT_712 responsible for the dissemination of *bla*NDM-1 genes within the genus of *Acinetobacter*. This study provides a framework for characterizing genetic diversity, evolutionary dynamics, molecular population distribution, and tracing of *A. pittii*, which has the potential to improve infection control strategies and public health policy.
    year: 2025
    image: "2025_cb_ap.png"
    journal: "Communications Biology"
    authors: "Zhu, S., Li, S., Wu, B. et al."
    doi: "10.1038/s42003-025-08156-y"
    link: "https://www.nature.com/articles/s42003-025-08156-y"

  - title: "Uncovering a cryptic *Streptococcus suis* endemic post-outbreak: Evidence of host switching to humans"
    abstract: >
      *Streptococcus suis* (*S. suis*) is a neglected and emerging pathogen that leads to severe economic losses in swine industry. Despite its epidemic potential, the zoonotic threat posed by *S. suis* remains underappreciated, even after the unprecedented Sichuan outbreak, which highlighted its ability to cause fatal human infections. Understanding of the dynamics and evolution of this pathogen in human populations is crucial for preventing future outbreaks. Our study revealed the emergence of highly pathogenic *S. suis* lineages in Zhejiang Province following the Sichuan outbreak, showing an increasingly specialized lifestyle that has persisted for nearly two decades. Phylogenetic analysis traced the zoonotic transmission of this pathogen back to a livestock lineage in the Netherlands prior to 1990, which eventually led to the Sichuan outbreak lineage in 2005 and its subsequent spread to Zhejiang the same year. Two independent evolved sub-lineages were identified in Zhejiang, suggesting a cryptic, regional endemicity following the Sichuan outbreak. Furthermore, the accumulation of lineage-specific resistance and metabolic acclimation after divergence from the Sichuan population suggested potential regional evolutionary shifts in *S. suis*. These new findings could help inform future intervention strategies and guide public health policies.
    year: 2025
    image: "2025_stoten_ss.png"
    journal: "Science of The Total Environment"
    authors: "Wang, S., Zhou, Y., Wang, Y. et al."
    doi: "10.1016/j.scitotenv.2024.178307"
    link: "https://www.sciencedirect.com/science/article/abs/pii/S0048969724084651"

  - title: "KleTy: integrated typing scheme for core genome and plasmids reveals repeated emergence of multi-drug resistant epidemic lineages in *Klebsiella* worldwide"
    abstract: >
      **Background**   Clinically important lineages in *Klebsiella*, especially those expressing multi-drug resistance (MDR), pose severe threats to public health worldwide. They arose from the co-evolution of the vertically inherited core genome and horizontal gene transfers by plasmids, which has not been systematically explored.   **Methods** We designed KleTy, which consists of dedicated typing schemes for both the core genome and plasmids in *Klebsiella*. We compared the performance of KleTy with many state-of-the-art pipelines using both simulated and real data.    **Results** Employing KleTy, we genotyped 33,272 *Klebsiella* genomes, categorizing them into 1773 distinct populations and predicting the presence of 87,410 plasmids from 837 clusters (PCs). Notably, *Klebsiella* is the center of the plasmid-exchange network within *Enterobacteriaceae*. Our results associated the international emergence of prevalent *Klebsiella* populations with only four carbapenem-resistance (CR) PCs, two hypervirulent PCs, and two hvCR-PCs encoding both carbapenemase and hypervirulence. Furthermore, we observed the ongoing international emergence of *bla*NDM, accompanied by the replacement of the previously dominant population, *bla*KPC-encoding HC1360_8 (CC258), during 2003–2018, with the emerging *bla*NDM-encoding HC1360_3 (CC147) thereafter. Additionally, expansions of hypervirulent carbapenem-resistant *Klebsiella pneumoniae* (hvCRKP) were evidenced in both populations, driven by plasmids of MDR-hypervirulence convergences.  **Conclusions** The study illuminates how the global genetic landscape of *Klebsiella* has been shaped by the co-evolution of both the core genome and the plasmids, underscoring the importance of surveillance and control of the dissemination of plasmids for curtailing the emergence of hvCRKPs.
    year: 2024
    image: "2024_gm_klety.png"
    journal: "Genome Medicine"
    authors: "Li, H., Liu, X., Li, S. et al."
    doi: "10.1186/s13073-024-01399-0"
    link: "https://link.springer.com/article/10.1186/s13073-024-01399-0"

  - title: "Epidemiological and molecular investigations of sequential outbreaks of *Campylobacter jejuni* infecting adults and schoolchildren in southeastern China, 2021-2022"
    abstract: >
      **Objectives**  To investigate cases of five *Campylobacter jejuni* outbreaks and describe laboratory characteristics of these infections.   **Methods** Whole-genome sequencing and conventional methods were combined to thoroughly investigate the outbreaks, and data of contemporaneous sporadic cases was included for comparison.    **Results** Seven sequence types (STs) of *C. jejuni* caused 83 cases, including ST9079 which recurred across 2 years. Trace-back investigation could not identify any food items of infection but detected identical campylobacters from food contacts. Phylogenetic analysis unveiled genetic closeness between outbreak strains and some concurrent sporadic strains, indicating local campylobacteriosis may not be wholly sporadic but rather a series of linked cases. Virulence genes disclosed species/case-specific signatures to differentiate outbreaks from truly non-outbreak strains. Resistance to fluoroquinolones and/or macrolides was prevalent (90.8%, 108/119), with a noteworthy portion exhibiting multidrug resistance (31.1%, 37/119). Five types of plasmids were harbored among outbreak isolates, of which one plasmid harboring anti-stress and resistant genes was rarely found in *C. jejuni*.  **Conclusions** This is the first reported sequential outbreak of *C. jejuni* in China. Our observations help to define the genomic landscape and antimicrobial resistance patterns of *Campylobacter*, emphasizing the need for a broader ‘One Health’ perspective to combat the threats posed by campylobacteriosis.
    year: 2024
    image: "2024_ijid_cj.jpg"
    journal: "International Journal of Infectious Diseases"
    authors: "Zhang, L., Shao, Y., Li, S. et al."
    doi: "10.1016/j.ijid.2024.107055"
    link: "https://www.sciencedirect.com/science/article/pii/S1201971224001255"

  - title: "Resurgence of *Mycoplasma* pneumonia by macrolide-resistant epidemic clones in China"
    abstract: >
      We found that EC2 showed 100% macrolide resistance because of the acquisition of the A2063G mutation in 23S rRNA. All three P1-2 strains isolated in Beijing in 2020 were derived from EC2, suggesting that EC2 could be responsible for the previously reported increase in MRMP frequencies in P1-2 strains detected in the country after 2018. Moreover, our analyses showed frequent, cryptic cross-regional *M pneumoniae* transmissions in both EC1 and EC2, and the Taiwan strains in 2019 showed that the EC2 MRMP strain had been spreading throughout China before 2020. PCR-based surveillance data also suggested an unusual increase in infections due to the P1-2 MRMP strain in north China from 2021 to 2022. Our findings suggest that without the restrictions enacted during the COVID-19 pandemic, the MRMP clones could have caused outbreaks across the country as early as 2020; thus, our findings provide evidence to guide future epidemic prevention and treatment in east Asia.
    year: 2024
    image: "2024_lm_mp.png"
    journal: "The Lancet Microbe"
    authors: "Li, H., Li, S., Yang, H. et al."
    doi: "10.1016/S2666-5247(23)00405-6"
    link: "https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(23)00405-6/fulltext"

  - title: "Multicenter Genomic Epidemiology Reveals Distinct Short-Term and Long-Term Crab Transmission Patterns in Zhejiang, China"
    abstract: >
      **Objectives**  To investigate the genomic epidemiology of carbapenem-resistant *Acinetobacter baumannii* (CRAB) in ICUs of municipal hospitals in Zhejiang province, and to elucidate its transmission patterns and determine its spread influencing factors.   **Methods** We conducted a multicenter genomic epidemiological analysis of 138 CRAB isolates collected from ICUs of tertiary hospitals across 8 prefecture-level cities in Zhejiang province between 2018 and 2023. Antimicrobial susceptibility testing and whole 49 genome sequencing were performed on these isolates. By integrating genomic data from the capital city in public database, we identified population structures and reconstructed transmission events to assess dynamic changes over time.    **Results** Between 2018 and 2023, CRAB isolates from tertiary hospitals in 9 prefecture-level cities in Zhejiang Province predominantly consisted of the ESL2.4 and ESL2.5 lineages. The profile of antimicrobial resistance genes was strongly associated with phylogenetic lineage. Short-term outbreaks identified using a 5-SNP threshold, demonstrated a significant correlation with geographical distances within 160 km. Temporal phylogenetic analysis revealed that the prolonged and sustained transmission of CRAB strains across ICUs in Zhejiang originating in the 2010s, coincided notably with the rapid development and enhancement of traffic and public transportation in the provincial capital starting in 2010.  **Conclusions** CRAB in ICUs from 9 hospitals in Zhejiang Province predominantly consisted of the ESL2.4 and ESL2.5 lineages. Geographical distance and transportation are critical factors influencing the transmission of CRAB outbreaks among ICUs in different cities. The provincial capital exerts a persistent and significant influence on non-capital cities beyond geographical constraints.
    year: 2025
    image: "2025_pp_ab.png"
    journal: "Preprint"
    authors: "Li, S., Wang, S., Pu, F. et al."
    doi: "10.2139/ssrn.5043757"
    link: "https://papers.ssrn.com/sol3/papers.cfm?abstract_id=5043757"

  - title: "Centralized industrialization of pork in Europe and America contributes to the global spread of *Salmonella enterica*"
    abstract: >
      *Salmonella enterica* causes severe food-borne infections through contamination of the food supply chain. Its evolution has been associated with human activities, especially animal husbandry. Advances in intensive farming and global transportation have substantially reshaped the pig industry, but their impact on the evolution of associated zoonotic pathogens such as *S. enterica* remains unresolved. Here we investigated the population fluctuation, accumulation of antimicrobial resistance genes and international serovar Choleraesuis transmission of nine pig-enriched *S. enterica* populations comprising more than 9,000 genomes. Most changes were found to be attributable to the developments of the modern pig industry. All pig-enriched salmonellae experienced host transfers in pigs and/or population expansions over the past century, with pigs and pork having become the main sources of *S. enterica* transmissions to other hosts. Overall, our analysis revealed strong associations between the transmission of pig-enriched salmonellae and the global pork trade.
    year: 2024
    image: "2024_nf_se.png"
    journal: "Nature Food"
    authors: "Li, H., Wu, Y., Feng, D. et al."
    doi: "10.1038/s43016-024-00968-1"
    link: "https://www.nature.com/articles/s43016-024-00968-1"

  - title: "Characterization of microbial community and antibiotic resistome in intra urban water, Wenzhou China"
    abstract: >
      The present study investigated the water quality index, microbial composition and antimicrobial resistance genes in urban water habitats. Combined chemicals testing, metagenomic analyses and qualitative PCR (qPCR) were conducted on 20 locations, including rivers from hospital surrounds (n = 7), community surrounds (n = 7), and natural wetlands (n = 6). Results showed that the indexes of total nitrogen, phosphorus, and ammonia nitrogen of hospital waters were 2–3 folds high than that of water from wetlands. Bioinformatics analysis revealed a total of 1,594 bacterial species from 479 genera from the three groups of water samples. The hospital-related samples had the greatest number of unique genera, followed by those from wetlands and communities. The hospital-related samples contained a large number of bacteria associated with the gut microbiome, including *Alistipes*, *Prevotella*, *Klebsiella*, *Escherichia*, *Bacteroides*, and *Faecalibacterium*, which were all significantly enriched compared to samples from the wetlands. Nevertheless, the wetland waters enriched bacteria from *Nanopelagicus*, *Mycolicibacterium* and *Gemmatimonas*, which are typically associated with aquatic environments. The presence of antimicrobial resistance genes (ARGs) that were associated with different species origins in each water sample was observed. The majority of ARGs from hospital-related samples were carried by bacteria from *Acinetobacter*, *Aeromonas* and various genera from *Enterobacteriaceae*, which each was associated with multiple ARGs. In contrast, the ARGs that were exclusively in samples from communities and wetlands were carried by species that encoded only 1 to 2 ARGs each and were not normally associated with human infections. The qPCR showed that water samples of hospital surrounds had higher concentrations of intI1 and antimicrobial resistance genes such as *tetA*, *ermA*, *ermB*, *qnrB*, *sul1*, *sul2* and other beta-lactam genes. Further genes of functional metabolism reported that the enrichment of genes associated with the degradation/utilization of nitrate and organic phosphodiester were detected in water samples around hospitals and communities compared to those from wetlands. Finally, correlations between the water quality indicators and the number of ARGs were evaluated. The presence of total nitrogen, phosphorus, and ammonia nitrogen were significantly correlated with the presence of *ermA* and *sul1*. Furthermore, intI1 exhibited a significant correlation with *ermB*, *sul1*, and *bla*SHV, indicating a prevalence of ARGs in urban water environments might be due to the integron intI1’s diffusion-promoting effect. However, the high abundance of ARGs was limited to the waters around the hospital, and we did not observe the geographical transfer of ARGs along with the river flow. This may be related to water purifying capacity of natural riverine wetlands. Taken together, continued surveillance is required to assess the risk of bacterial horizontal transmission and its potential impact on public health in the current region.
    year: 2023
    image: "2023_fm_wzwater.png"
    journal: "Frontiers in Microbiology"
    authors: "Ye, S., Li, S., Su, C. et al."
    doi: "10.3389/fmicb.2023.1169476"
    link: "https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1169476/full"

  - title: "The correlation between dysfunctional intestinal flora and pathology feature of patients with pulmonary tuberculosis"
    abstract: >
      Recent studies have provided insights into the important contribution of gut microbiota in the development of Pulmonary Tuberculosis (PTB). As a chronic consumptive infectious disease, PTB involves many pathological characteristics. At present, research on intestinal flora and clinical pathological Index of PTB is still rare. We performed a cross-sectional study in 63 healthy controls (HCs) and 69 patients with untreated active PTB to assess the differences in their microbiota in feces via 16S rRNA gene sequencing. Significant alteration of microbial taxonomic and functional capacity was observed in PTB as compared to the HCs. The results showed that the alpha diversity indexes of the PTB patients were lower than the HCs (P<0.05). Beta diversity showed differences between the two groups (P<0.05). At the genus level, the relative abundance of Bacteroides, Parabacteroides and Veillonella increased, while Faecalibacterium, Bifidobacterium, Agathobacter and CAG-352 decreased significantly in the PTB group, when compared with the HCs. The six combined genera, including *Lactobacillus*, *Faecalibacterium*, *Roseburia*, *Dorea*, *Monnoglobus* and [Eubacterium]_ventriosum_group might be a set of diagnostic biomarkers for PTB (AUC=0.90). Besides, the predicted bacterial functional pathway had a significant difference between the two groups (P<0.05), which was mainly related to the nutrient metabolism pathway. Significant alterations in the biochemical index were associated with changes in the relative abundance of specific bacteria, the short chain fatty acid (SCFA)-producing bacteria enriched in HCs had a positively correlated with most of the biochemical indexes. Our study indicated that the gut microbiota in PTB patients was significantly different from HCs as characterized by the composition and metabolic pathway, which related to the change of biochemical indexes in the PTB group. It was hypothesized that the abovementioned changes in the gut microbiota could exert an impact on the clinical characteristics of PTB through the regulation of the nutrient utilization pathway of the host by way of the gut-lung axis.
    year: 2023
    image: "2023_fcim_ptb.png"
    journal: "Frontiers in Cellular and Infection Microbiology"
    authors: "Ye, S., Wang, L., Li, S. et al."
    doi: "10.3389/fcimb.2022.1090889"
    link: "https://www.frontiersin.org/journals/cellular-and-infection-microbiology/articles/10.3389/fcimb.2022.1090889/full"
---

This page contains a list of my major academic publications.
